Changelog.md

```diff
@@ -18,6 +18,7 @@ Bugfixes:
  * Commandline Interface: It is no longer possible to specify both ``--optimize-yul`` and ``--no-optimize-yul`` at the same time.
  * SMTChecker: Fix encoding of side-effects inside ``if`` and ``ternary conditional``statements in the BMC engine.
  * SMTChecker: Fix false negative when a verification target can be violated only by trusted external call from another public function.
+ * Yul Optimizer: Optimized IR being unnecessarily passed through Yul optimizer again before bytecode generation.
```

Comment:
```suggestion
 * Yul Optimizer: Fix optimized IR being unnecessarily passed through the Yul optimizer again before bytecode generation.
```
or
```suggestion
 * Yul Optimizer: Prevent optimized IR from being unnecessarily passed through the Yul optimizer again before bytecode generation.
```
?

Reply:
Done. I went with the first variant.

Reply:
Hmm... I just pushed the change (a3c5b6cc6910591d9c9a245c3f50b0bee81c1b37) but github is still showing the old commit (8d56c3bce2564e2e9fc2473fa5d3850d3bd5a37d).